src/Nethermind/Nethermind.Facade/Filters/LogFinder.cs

```diff
@@ -63,6 +63,9 @@ public class LogFinder : ILogFinder
 
         public IEnumerable<FilterLog> FindLogs(LogFilter filter, CancellationToken cancellationToken = default)
         {
+            if (filter == null)
+                return Enumerable.Empty<FilterLog>();
```

Comment:
yes - after this change it might be possible (though, highly unlikely):
`        
public T? GetFilter<T>(int filterId) where T : FilterBase => _filters.TryGetValue(filterId, out var filter)
                ? filter as T
                : null;
`
It's more of a future bullet-proofing the code.

Reply:
Yes but isn't this null handled eslewhere?

Reply:
Removed following the change to bridge (TryGetLogs implementation).